src/rdo.rs

```diff
@@ -159,13 +159,12 @@ pub fn get_lambda_sqrt(fi: &FrameInvariants) -> f64 {
   q0 * (std::f64::consts::LN_2 / 6.0).sqrt()
 }
 
-// Compute the rate-distortion cost for an encode
-fn compute_rd_cost(
+// Compute the distortion cost for an encode
+fn compute_distortion(
   fi: &FrameInvariants, fs: &FrameState, w_y: usize, h_y: usize,
-  is_chroma_block: bool, bo: &BlockOffset, bit_cost: u32,
+  is_chroma_block: bool, bo: &BlockOffset,
   luma_only: bool
-) -> f64 {
-  let lambda = get_lambda(fi);
+) -> u64 {
 
   // Compute distortion
   let po = bo.plane_offset(&fs.input.planes[0].cfg);
```

Comment:
The if/elseif/else block starting on the next line should really be replaced by a match.